client/src/components/widgets/charts/MultiLinePlot.vue

```diff
@@ -41,18 +40,43 @@
       fill: 'none',
       strokeWidth: 2.5,
       strokeColor: Colors.NODES.DEFAULT,
-      strokeColorOther: Colors.NODES.OTHER,
-      strokeColorAggregate: Colors.NODES.AGGREGATE,
     },
     label: {
       text: Colors.LABELS.LIGHT,
     },
   };
 
+  const AGGREGATE_STYLE = {
+    node: {
+      fill: Colors.NODES.AGGREGATE,
+    },
+    edge: {
+      strokeColor: Colors.NODES.AGGREGATE,
+    },
+  };
+
+  const OTHER_STYLE = {
+    node: {
+      fill: Colors.NODES.OTHER,
+    },
+    edge: {
+      strokeColor: Colors.NODES.OTHER,
+    },
+  };
+
+  const mergeStyle = modifyingStyle => {
+    if (modifyingStyle) {
+      return merge(cloneDeep(DEFAULT_STYLE), modifyingStyle);
+    }
+
+    return DEFAULT_STYLE;
+  };
+
 @Component
   export default class VariablePanel extends Vue {
     @Prop({ default: 'Title' }) title: string;
     @Prop({ default: () => [[]] }) data: any;
+    @Prop({ default: () => [[]] }) dataAggregate: any;
```

Comment:
Why not adding the style inside the data object? to avoid different `data` prop in the future.
The `data` type can include a metadata property including `name`, `style`, etc.

Reply:
i like this idea

Reply:
I see that you created a new prop instead of extending the data prop. That works as well.